source/lmp/pair_deepmd.cpp

```diff
@@ -1049,8 +1165,21 @@ void PairDeepMD::settings(int narg, char **arg) {
       eps_v = strtof(arg[iarg + 1], NULL);
 #endif
       iarg += 2;
+    } else if (string(arg[iarg]) == string("virtual_len")) {
```

Comment:
Please add documentation for new paramters

Reply:
We don't know where the documentation should be put. For example, we didn't find the documentation for `relative`. Please give us an example.

Reply:
https://github.com/deepmodeling/deepmd-kit/blob/7020b345d3973b6de73431938315387aa2cedde7/doc/third-party/lammps-command.md?plain=1#L72

Reply:
OK. We have added documentations for spin parameters.